<ui version="4.0" >
 <class>clockConfig</class>
 <widget class="QWidget" name="clockConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>381</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="1" column="1" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Font style:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>plainClockFont</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="KFontComboBox" name="plainClockFont" />
     </item>
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>78</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="2" colspan="2" >
    <widget class="QCheckBox" name="plainClockFontBold" >
     <property name="toolTip" >
      <string>Check if you want the font in bold</string>
     </property>
     <property name="whatsThis" >
      <string>When this is checked, the clock font will be bold.</string>
     </property>
     <property name="text" >
      <string>&amp;Bold</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2" >
    <widget class="QCheckBox" name="plainClockFontItalic" >
     <property name="toolTip" >
      <string>Check if you want the font in italic</string>
     </property>
     <property name="whatsThis" >
      <string>When this is checked, the clock font will be in italic.</string>
     </property>
     <property name="text" >
      <string>&amp;Italic</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>409</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Font color:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>useThemeColor</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2" >
    <widget class="QRadioButton" name="useThemeColor" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Use current desktop theme color</string>
     </property>
     <property name="whatsThis" >
      <string>This is default. The clock will get its font color from the current desktop theme.</string>
     </property>
     <property name="text" >
      <string>Use theme color</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QRadioButton" name="useCustomColor" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Choose your own font color</string>
       </property>
       <property name="whatsThis" >
        <string>When checked you can choose a custom color for the clock font by clicking on the color widget on the right.</string>
       </property>
       <property name="text" >
        <string>Use custom color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="plainClockColor" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Color chooser</string>
       </property>
       <property name="whatsThis" >
        <string>Click on this button and the KDE standard color dialog will show. You can then choose the new color you want for your clock.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_5" >
     <item>
      <widget class="QCheckBox" name="showDate" >
       <property name="toolTip" >
        <string>Display the date of the day</string>
       </property>
       <property name="whatsThis" >
        <string/>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="9" column="3" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" >
      <widget class="QCheckBox" name="showDay" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Display day of the week</string>
       </property>
       <property name="whatsThis" >
        <string>Add the day of the week to the date display.</string>
       </property>
       <property name="text" >
        <string>Show day of the &amp;week</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" >
      <widget class="QCheckBox" name="showYear" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Display the current year</string>
       </property>
       <property name="whatsThis" >
        <string>Add the year to the date string.</string>
       </property>
       <property name="text" >
        <string>Show &amp;year</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="3" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QCheckBox" name="showTimeZone" >
       <property name="toolTip" >
        <string>Display the time zone name</string>
       </property>
       <property name="whatsThis" >
        <string>Display the time zone name under the time.</string>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="12" column="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <widget class="QCheckBox" name="secondsCheckbox" >
       <property name="toolTip" >
        <string>Show the seconds</string>
       </property>
       <property name="whatsThis" >
        <string>Check this if you want to show the seconds.</string>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="13" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label_6" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Font</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <widget class="QLabel" name="label_7" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Information</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer name="horizontalSpacer_9" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="1" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Show time zone:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>showTimeZone</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="1" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Show seconds:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>secondsCheckbox</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Show date:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>showDate</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontComboBox</class>
   <extends>KComboBox</extends>
   <header>kfontcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>showDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>showDay</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>124</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>347</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>showYear</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>91</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>407</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>plainClockColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>155</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel" >
     <x>258</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
